client/src/app/pages/assessment/components/assessment-wizard/assessment-wizard-modal.tsx

```diff
@@ -0,0 +1,44 @@
+import { Modal, ModalVariant } from "@patternfly/react-core";
+import React, { FunctionComponent } from "react";
+import { AssessmentWizard } from "./assessment-wizard";
+import { useFetchAssessmentById } from "@app/queries/assessments";
+
+interface AssessmentModalProps {
+  isOpen: boolean;
+  onRequestClose: () => void;
+  assessmentId: number;
+}
+
+const AssessmentModal: FunctionComponent<AssessmentModalProps> = ({
+  isOpen,
+  onRequestClose,
+  assessmentId,
+}) => {
+  const { assessment, isFetching, fetchError } =
+    useFetchAssessmentById(assessmentId);
+
+  return (
+    <>
+      {isOpen && (
+        <Modal
+          isOpen={isOpen}
+          onClose={onRequestClose}
+          showClose={false}
```

Comment:
```suggestion
          showClose={isFetching || !!fetchError}
```

this seems to fix the focus trap issue. it is an issue when you have an error/loader rather than the wizard.